seqr/migrations/0064_auto_20190826_1748.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-08-26 17:48
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seqr', '0063_auto_20190806_1801'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='varianttag',
+            unique_together=set([]),
+        ),
+        migrations.RemoveField(
+            model_name='variantnote',
+            name='saved_variant',
+        ),
+        migrations.AddField(
+            model_name='variantnote',
+            name='saved_variant',
+            field=models.ManyToManyField(to='seqr.SavedVariant'),
+        ),
+        migrations.RemoveField(
```

Comment:
this almost definitely effectively deletes all the tags/ notes in seqr. Lets discuss in person